heartbeat/monitors/active/http/check_test.go

```diff
@@ -0,0 +1,106 @@
+package http
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/elastic/beats/libbeat/common/match"
+)
+
+func TestCheckBody(t *testing.T) {
+
+	var matchTests = []struct {
+		description string
+		body        string
+		patterns    []match.Matcher
```

Comment:
as these are patterns, how about `[]string`, so to make the table a little more readable?

Reply:
If it isn't obvious I'm still very new to writing go :) Does this look a bit more readable now? https://github.com/elastic/beats/pull/6539/commits/f63257b1f50825a3f51a393ba56f502b2827bdc4 Or is there maybe an even cleaner way to do it?